DevEdu.API/DevEdu.Business.Tests/Data/CourseData.cs

```diff
@@ -12,6 +8,152 @@ public class CourseData
         public CourseData()
         {
 
+        }
+        public const int ExpectedAffectedRows = 1;
+        public const int CourseId = 1;
```

Comment:
it's a bad practice to have such values as consts
it's better to declare them in each test method separately

Reply:
+